frontend/src/hooks/usePollenItems.js

```diff
@@ -0,0 +1,18 @@
+import {useEffect, useState} from "react";
+import * as apiservice from "../service/apiservice"
+
+export default function usePollenItems() {
+    const [pollenItems, setPollenItems] = useState([]);
+
+    useEffect(() => {
+        loadData()
+    }, [])
+
+    const loadData = () => {
```

Comment:
i would rename the function to load PollenData or something that more describes what happens